libutils/sequence.c

```diff
@@ -90,6 +90,12 @@ static void ExpandIfNeccessary(Seq *seq)
     }
 }
 
+int StrCmpWrapper(const void *subject, const void *anchor, void *user_data)
```

Comment:
I don't think subject and anchor are good names here, just use the same as strcmp.